src/commands/uplink/DataHourMul.ts

```diff
@@ -4,18 +4,21 @@
 /* eslint-disable no-restricted-syntax */
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
 import Command from '../../Command.js';
-import DataDayMul from './DataDayMul.js';
+import GetCurrentMul from './GetCurrentMul.js';
 import {getSecondsFromDate, getDateFromSeconds} from '../../utils/time.js';
-import decodeExtendedBytes from '../../utils/decodeExtendedBytes.js';
-import encodeExtendedBytes from '../../utils/encodeExtendedBytes.js';
+import CommandBinaryBuffer from '../../CommandBinaryBuffer.js';
 
 
 const COMMAND_ID = 0x17;
 const COMMAND_TITLE = 'DATA_HOUR_MUL';
-const CHANNELS_MASK = [0x01, 0x02, 0x04, 0x08];
 
+// date 2 bytes, hour 1 byte, channels - 1 byte, so max channels = 4
+// max hours diff - 7 (3 bit value)
+// 4 + (4 channels * 3 bytes of hour values) + (4 * 2 bytes of diff * 7 max hours diff)
+const COMMAND_BODY_MAX_SIZE = 104;
```

Comment:
something is not right

`4+4*3+4*2*7 = 72`

Reply:
fixed